lib/filewatch/bootstrap.rb

```diff
@@ -0,0 +1,91 @@
+# encoding: utf-8
+require "rbconfig"
+require "pathname"
+
+## Common setup
+#  all the required constants and files
+#  defined in one place
+module FileWatch
+  HOST_OS_WINDOWS = (RbConfig::CONFIG['host_os'] =~ /mswin|mingw|cygwin/) != nil
```

Comment:
FYI in Logstash https://github.com/elastic/logstash/blob/5e49b5d9cbb0a66482555a2ae57c0e89379f3923/lib/bootstrap/environment.rb#L45-L47 this is how we test for the windows platform. Should we be consistent?
```
    def windows?
      ::Gem.win_platform?
    end
```

Reply:
Thanks for pointing this out. It is a leftover from filewatch being standalone.

Will refactor this to use logstash/environment.